origin-discovery/src/apollo/listingMetadata.js

```diff
@@ -0,0 +1,38 @@
+// how frequently featured/hidden listings list updates
```

Comment:
nit: I think our convention is that we don't use cameCase for names of files. So please rename to listing-metadata.js (or simply metadata.js, up to you). BTW I like the name metadata much better than info. good call on that ! :)

nit2: Add a high level comment at top of the file explaining what this does.
It loads and caches featured and hidden listing ids metadata.

Reply:
thanks 👍